grakn-engine/src/main/java/ai/grakn/engine/postprocessing/UpdatingInstanceCountTask.java

```diff
@@ -41,31 +45,92 @@
  *
  * @author fppt
  */
-public class UpdatingInstanceCountTask extends AbstractLockingTask {
-
-    public static final String LOCK_KEY = "/updating-instance-count-lock";
+public class UpdatingInstanceCountTask implements BackgroundTask {
+    public static final RedisConnection redis = RedisConnection.getConnection();
```

Comment:
We should try and handle it a bit better. If it throws a runtime exception within a task all of the `UpdatingInstanceCountTask`s will be marked as `FAILED`, which is fine, but we need to make sure the error is descriptive enough that people know there is an issue with Redis